packages/slice-machine/src/apiClient.ts

```diff
@@ -47,3 +49,9 @@ export const sendTrackingOnboarding = (
     | OnboardingContinueWithVideoEvent
 ): Promise<AxiosResponse<TrackingReviewResponse>> =>
   axios.post(`/api/tracking/onboarding`, onboardingEvent, defaultAxiosConfig);
+
+/** Version info Route **/
+
+export function getVersionInfo(): Promise<AxiosResponse<VersionInfo>> {
```

Comment:
Good but I think we don't need this route

Reply:
if we can fix the env we won't